webapp/src/components/ImageFieldEditor.tsx

```diff
@@ -186,6 +186,24 @@ export class ImageFieldEditor<U extends pxt.Asset> extends React.Component<Image
             assets = assets.map(t => (t.type !== this.asset.type || t.id !== this.asset.id) ? t : assetToGalleryItem(this.getValue()))
         }
 
+        if (filterString) {
+            assets.forEach(a => {
+                if (!a.meta.tags && this.options) {
+                    a.meta.tags = this.blocksInfo.apis.byQName[a.id]?.attributes.tags?.split(" ") || [""];
```

Comment:
```suggestion
                    a.meta.tags = this.blocksInfo.apis.byQName[a.id]?.attributes.tags?.split(" ") || [];
```
nit; should be fine to just do []?